### PR TITLE
Fix build for x86_64

### PR DESCRIPTION
**Pull request by BachToTheFuture**
BeAccessible was broken for x86_64, so I fixed up some things and it should work as shown in the screenshot in README.

Speaking of the README, I reformatted it to Markdown, added screenshots, and added build directions 😄

### CODE REVIEW

**Comment by scottmc** [comment]
@waddlesplash or @owenca can you review this one?

File: src/BeAccessibleWindow.cpp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #include <FilePanel.h>
 #include <Path.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/BeAccessibleWindow.cpp]: Sort as in comment 1.

File: src/BeAccessibleApp.cpp [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #include <Path.h>
 #include <Alert.h>
 #include <ListView.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/BeAccessibleApp.cpp]: While you're there, alpha-sort the includes. Also it'd be nice to group and sort headers:
posix -> native -> shared private -> app private
Add an empty line between the groups.

File: src/BeAccessibleWindow.h [hunk 3]
```diff
@@ -8,6 +8,7 @@
 #endif
 #include <Path.h>
 #include <Menu.h>
+#include <StorageKit.h>
```
Comment (humdingerb) [inline src/BeAccessibleWindow.h:12]: Sort as in comment 1.

File: src/ColumnDesignView.cpp [hunk 4]
```diff
@@ -6,7 +6,7 @@
 #include <RadioButton.h>
 #include <CheckBox.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/ColumnDesignView.cpp]: Sort as in comment 1.

File: src/Constants.h [hunk 5]
```diff
@@ -1,8 +1,8 @@
 #ifndef CONSTANTS_H
 #define CONSTANTS_H
 
-#include <iostream.h>
-#include <fstream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/Constants.h:4]: Sort as in comment 1.

File: src/DBListView.cpp [hunk 6]
```diff
@@ -1,4 +1,4 @@
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/DBListView.cpp:1]: Sort as in comment 1.

File: src/DBTableTabView.cpp [hunk 7]
```diff
@@ -5,7 +5,7 @@
 #endif
 #include <ListView.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/DBTableTabView.cpp]: Sort as in comment 1.

File: src/GridCell.cpp [hunk 8]
```diff
@@ -1,6 +1,6 @@
 #include <Window.h>
 #include <InterfaceDefs.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/GridCell.cpp:1]: Sort as in comment 1.

File: src/DBTabView.cpp [hunk 9]
```diff
@@ -4,7 +4,7 @@
 #include "DBTabView.h"
 #endif
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/DBTabView.cpp]: Sort as in comment 1.

File: src/DelimiterView.cpp [hunk 10]
```diff
@@ -1,5 +1,5 @@
 #include <Message.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/DelimiterView.cpp:1]: Sort as in comment 1.

File: src/GridRow.cpp [hunk 11]
```diff
@@ -5,7 +5,7 @@
 #include <MenuItem.h>
 #include <Alert.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/GridRow.cpp]: Sort as in comment 1.

File: src/GridView.cpp [hunk 12]
```diff
@@ -1,5 +1,5 @@
 #include <Window.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/GridView.cpp:1]: Sort as in comment 1.

File: src/ImportFileView.cpp [hunk 13]
```diff
@@ -3,7 +3,7 @@
 #include <Message.h>
 #include <CheckBox.h>
 #include <Alert.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/ImportFileView.cpp:1]: Sort as in comment 1.

File: src/ImportFileWindow.cpp [hunk 14]
```diff
@@ -1,6 +1,6 @@
 #include <Message.h>
-#include <iostream.h>
-#include <fstream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/ImportFileWindow.cpp:1]: Sort as in comment 1.

File: src/PasteTableView.cpp [hunk 15]
```diff
@@ -2,7 +2,7 @@
 #include <Button.h>
 #include <RadioButton.h>
 #include <Box.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/PasteTableView.cpp]: Sort as in comment 1.

File: src/PasteTableWindow.cpp [hunk 16]
```diff
@@ -1,6 +1,6 @@
 #include <Application.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/PasteTableWindow.cpp]: Sort as in comment 1.

File: src/PreviewView.cpp [hunk 17]
```diff
@@ -1,4 +1,4 @@
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/PreviewView.cpp:1]: Sort as in comment 1.

File: src/RowSummaryView.cpp [hunk 18]
```diff
@@ -1,6 +1,6 @@
 #include <Region.h>
 #include <Window.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/RowSummaryView.cpp:1]: Sort as in comment 1.

File: src/SQLiteManager.cpp [hunk 19]
```diff
@@ -1,5 +1,5 @@
 #include <Alert.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/SQLiteManager.cpp:1]: Sort as in comment 1.

File: src/TableDataRow.cpp [hunk 20]
```diff
@@ -4,7 +4,7 @@
 #include <PopUpMenu.h>
 #include <MenuItem.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/TableDataRow.cpp]: Sort as in comment 1.

File: src/TableDesignRow.cpp [hunk 21]
```diff
@@ -5,7 +5,7 @@
 #include <PopUpMenu.h>
 #include <MenuItem.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/TableDesignRow.cpp]: Sort as in comment 1.

File: src/TableWindow.cpp [hunk 22]
```diff
@@ -1,6 +1,6 @@
 #include <Application.h>
 #include <String.h>
-#include <iostream.h>
+#include <iostream>
```
Comment (humdingerb) [inline src/TableWindow.cpp:1]: Sort as in comment 1.

**Review by humdingerb — CHANGES_REQUESTED** [review]
I don't have a 64bit installation at hand, so other mentors would have to test.

**Comment by scottmc** [comment]
The readme mentions two code bases, I haven't looked that closely, but it appears we only put one into the repo? Research this and perhaps update the readme to reflect that the sqlite3 version of the original code is what is in the repo (if that is the case, I assume it is since it's using sqlite3.h).

**Comment by BachToTheFuture** [comment]
Ok... I can't count. I'll just stop numbering things :)

**Comment by BachToTheFuture** [comment]
@humdingerb Done! That was painful, but got it all done 😄 

@scottmc As for the code bases, I think you're right: the README does mention that the src directory is the sqlite3 version. I'll specify this in the readme.

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Well, you didn't have to do a separate commit for every header re-ordering... :)

**Comment by BachToTheFuture** [comment]
@humdingerb could you check my GCI task?

**Comment by BachToTheFuture** [comment]
@scottmc Is this what you expected?

**Comment by owenca** [comment]
@bach5000 One git command I found useful is `git commit —amend` if you like to commit often for small incremental changes. You can also squash your commits locally before pushing.

**Pull request merged**